src/app/mod.rs

```diff
@@ -98,6 +98,7 @@ impl<'a, 'b> App<'a, 'b> {
     /// [`crate_version!`]: ./macro.crate_version!.html
     /// [`App::author`]: ./struct.App.html#method.author
     /// [`App::version`]: ./struct.App.html#method.author
+    #[deprecated(since="2.14.1", note="Can never work; use explicit App::author() and App::version() calls instead")]
```

Comment:
Not quite sure which version should be put here, as it should be the next released one (which is either gonna be 2.14.1 or 2.15.0)